cypress-tests-v2/cypress/e2e/configs/Payment/Commons.js

```diff
@@ -964,7 +964,11 @@ export const connectorDetails = {
       Response: {
         status: 400,
         body: {
-          error: "Json deserialize error: invalid card number length",
+          error: {
+            error_type: "invalid_request",
+            message: "Json deserialize error: invalid card number length at line 1 column 638",
```

Comment:
Applies: here and everywhere

In the message, I suggest you not to include `at line <line number> column <column nunber>` as it is bound to vary request to request.

I believe we already have a validation added in `commands.js` that addresses where we validate this by verifying whether the content in the message is included in the response that we get.